equinix/fabric_mapping_helper.go

```diff
@@ -252,6 +264,27 @@ func locationToFabric(locationList []interface{}) v4.SimplifiedLocation {
 	return sl
 }
 
+func virtualdeviceToFabric(virtualdeviceList []interface{}) v4.VirtualDevice {
```

Comment:
I see that this method isn't parsing all the different parts of what could be sent in the virtual device according to our API spec. Just type and uuid.